### DISCUSSION TITLE
번아웃 극복기

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#441] [5. 회고]
원래 이 글을 어젯밤에 쓰고 올리려고 했는데 글 쓰다가 잠들어버렸습니다........
정신 차려보니까 아침이더라구요. 하하
[게시글 링크](https://amaran-th.github.io/%EC%A3%BC%EC%A0%80%EB%A6%AC%EC%A3%BC%EC%A0%80%EB%A6%AC/[%ED%9A%8C%EA%B3%A0]%20%EB%B2%88%EC%95%84%EC%9B%83%20%EA%B7%B9%EB%B3%B5%EA%B3%BC%20%EA%B0%9C%EB%B0%9C%EC%9E%90%EB%A1%9C%EC%84%9C%EC%9D%98%20%EA%B2%B0%EC%8B%AC/)

**Comment by shkisme** [comment]
아마란스 화이팅!! 늘 그래왔듯 너는 잘 극복할거라 믿어 의심치 않음!

다들 그동안 아마란스한테 좋은 영향 많이 받은 만큼 다들 많은 도움 줄 수 있을거야
또 언제나 내가 응원하고 있다 🚀

**Reply by amaran-th** [reply]
🥹🥹🥹🥹감동이야.... 
이런 격려까지 받았는데 늘어져 있을 순 없지
힘내볼게😁

**Comment by stopmin** [comment]
세연님 뽜이팅🔥🔥
뭘 하든 성공하실겁니당!!

**Reply by amaran-th** [reply]
감사합니다❤️❤️
버디님도 뭐든 잘 해내실 거에요👍
